<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackView</class>
 <widget class="QWidget" name="PlaybackView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxFile">
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="PlaybackFileView" name="widgetPlaybackFile" native="true">
            <layout class="QGridLayout" name="gridLayout_5"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxDevice">
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QWidget" name="widgetSerialDevice" native="true">
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Interface:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxInterface">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QGridLayout" name="gridLayoutOutput">
               <item row="0" column="0">
                <widget class="QWidget" name="widget" native="true"/>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>78</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonStartPlayback">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStopPlayback">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaybackFileView</class>
   <extends>QWidget</extends>
   <header>gui/PlaybackFileView.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonStartPlayback</sender>
   <signal>clicked()</signal>
   <receiver>PlaybackView</receiver>
   <slot>startPlayback()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStopPlayback</sender>
   <signal>clicked()</signal>
   <receiver>PlaybackView</receiver>
   <slot>stopPlayback()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxInterface</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PlaybackView</receiver>
   <slot>changeInterface(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startPlayback()</slot>
  <slot>stopPlayback()</slot>
  <slot>changeInterface(int)</slot>
 </slots>
</ui>
